ui/scss/component/_notification.scss

```diff
@@ -162,6 +169,30 @@ $contentMaxWidth: 60rem;
   }
 }
 
+.notification__reactions {
```

Comment:
### [Enhancement] Padding to align with avatar
It would be nice if we can pad the left side of the reactions area to align with the other text (i.e. right edge of avatar). 
![image](https://user-images.githubusercontent.com/64950861/130573059-cb39fdd1-2942-4f2c-bce1-a2590bf1b0ae.png)